pkg/api/client/client.go

```diff
@@ -10,6 +10,7 @@ import (
 	"net/url"
 	"path"
 
+	"github.com/gliderlabs/ssh"
```

Comment:
This package shouldn't be imported here because it is specific to ssh service.

Reply:
The reason of this import is because of the func RecordSession(uid string, waitingString string, pty ssh.Pty, opts models.ConfigOptions)